data/json/mapgen/lab/lab_surface/lab_surface_big_z5.json

```diff
@@ -0,0 +1,169 @@
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "........................................................................................................................",
+        "..........._________________________________................................_________________________________...........",
+        "..........._____________________AA__________................................__________AA_____________________...........",
+        "...........__D__________________pp__________................................__________pp__________________D__...........",
+        "..........._____________________pp__________................................__________pp_____________________...........",
+        "..........._____________________aa__________................................__________aa_____________________...........",
+        "...........____________>_>__________________................................__________________>_>____________...........",
+        "..........._________________________________................................_________________________________...........",
+        "..........._________________________________................................_________________________________...........",
+        "..........._________________________________................................_________________________________...........",
+        "..........._________________________________................................_________________________________...........",
+        "..........._________________________________................................_________________________________...........",
+        "...........___DD_____________________  DD___................................___DD_____________________  DD___...........",
```

Comment:
I don't think rooftop vending machines was intentional.

![image](https://user-images.githubusercontent.com/11464/54487021-1a23ae00-4845-11e9-9ad4-5238cc0bc002.png)

From looking at your palette, you've got `"f_satellite"` mapped to `D`, but you've also got the vending machine attribute mapped to `D`.

```
...
"furniture": {
      "D": "f_satellite",
      ...
    },
...
    "vendingmachines": { "D": { "item_group": "vending_drink" }, "F": { "item_group": "vending_food" } }
```
The result is what should probably be clusters of satellites on the roof are instead clusters of vending machines. Made me laugh though. 👍

Reply:
Rooftop access is so easy because that's where people go to dangle their legs over the edge and enjoy a refreshing Nuka-Cola